tests/integration/studies_blueprint/test_get_studies.py

```diff
@@ -454,6 +454,16 @@ def test_study_listing(
         study_map = res.json()
         assert not all_studies.intersection(study_map)
         assert all(map(lambda x: pm(x) in [PublicMode.READ, PublicMode.FULL], study_map.values()))
+        # test pagination
+        res = client.get(
+            STUDIES_URL,
+            headers={"Authorization": f"Bearer {john_doe_access_token}"},
+            params={"pageNb": 1, "pageSize": 2},
+        )
+        if len(study_map) > 2:
```

Comment:
I think you should remove this from your test as it's a bit confusing :)
Same goes for every other occurence

Reply:
yes, I made a generalization for the testing.